browser/brave_ads/BUILD.gn

```diff
@@ -0,0 +1,26 @@
+import("//brave/build/config.gni")
+import("//brave/components/brave_ads/browser/buildflags/buildflags.gni")
+
+source_set("brave_ads") {
+  # Remove when https://github.com/brave/brave-browser/issues/10639 is resolved
+  check_includes = false
```

Comment:
Do we still need these?

Reply:
this should have been removed @moritzhaller